examples/email-recovery/src/pages/index.tsx

```diff
@@ -0,0 +1,243 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+import { getWebAuthnAttestation, TurnkeyClient } from "@turnkey/http";
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import * as React from "react";
+import { useState } from "react";
+import { Recovery } from "@/components/Recovery";
+
+/**
+ * Type definition for the server response coming back from `/api/initRecovery`
+ */
+type InitRecoveryResponse = {
+  userId: string;
+  organizationId: string;
+};
+
+/**
+ * Type definitions for the form data (client-side forms)
+ */
+type RecoverUserFormData = {
+  recoveryBundle: string;
+  authenticatorName: string;
+};
+type InitRecoveryFormData = {
+  targetPublicKey: string;
+  email: string;
+};
+
+// All algorithms can be found here: https://www.iana.org/assignments/cose/cose.xhtml#algorithms
+// We only support ES256 and RS256, which are listed here
+const es256 = -7;
+const rs256 = -257;
+
+// This constant designates the type of credential we want to create.
+// The enum only supports one value, "public-key"
+// https://www.w3.org/TR/webauthn-2/#enumdef-publickeycredentialtype
+const publicKey = "public-key";
+
+const generateRandomBuffer = (): ArrayBuffer => {
+  const arr = new Uint8Array(32);
+  crypto.getRandomValues(arr);
```

Comment:
where is this `crypto` import coming from?

Reply:
This is the web crypto namespace, automatically available on the window object in browser environments (same as `document` or `navigator`)